tests/xen/test_ring0.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import secrets
+import time
+from typing import Optional
+import pytest
+
+from lib.commands import SSHCommandFailed
+from lib.host import Host
+from lib.vm import VM
```

Comment:
Needs to be reordered. Straight imports first, alphabetically. Then a blank line. Then "from xxx" imports, alphabetically.

Reply:
As Yann indicated above, I forgot about the split between external and internal libs, too.

Reply:
Stack Overflow most voted answer pushes for sorting, and that's also the convention we tend to use.

https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

Reply:
I thought you were originally refering to `pytest` not being sorted :wink:

Reply:
Fixed.